Initial PR (#1)
greenelab/nature_news_manuscript: content/04.results.md

## Results

![
**Data and Processing Pipeline Overview**
Panel A, left, depicts an example news article and the type of data extracted from the text.
Orange highlighted text depicts all named entities identified as either an organization, country, state, or province by the coreNLP pipeline.
The coreNLP pipeline also extracts all quotes and associated speakers.
A custom script described in section [Methods] identifies all citations.
Panel B charts the analyses done on the extracted names and locations from the news, research articles, and letters published by Nature.
Panel C shows the types and amounts of articles that we have used for analyses.
](https://github.com/nrosed/nature_news_disparities/raw/09df81eac6d8450127552f0b9dd3a6c1212b10eb/figure_notebooks/tmp_files/fig1_tmp/fig1_main.png "Figure 1"){#fig:fig1 width=6in}

> ### Data Collected

Reviewer: ```suggestion
### Creation of an Annotated News Dataset
```

The extracted author affiliations from both data sources were mapped to a country using OpenStreetMap.
Similarly, author names were uniformly processed and then used to predict both gender and name origin.

When considering all Nature articles, news and research, across all time, we find significant variability in the number of articles by type (Figure @fig:fig1 b).
The top three observed article frequencies are research (including letters and research), news, and news-and-views.
Since Nature merged letters and research articles in 2019, we combine them in our analysis.
The changing classification of article types may also explain temporal changes in news articles.
Over time, the frequency of news-related articles is decreasing; however, more specific article types are increasing.
We observe this in the introduction of two new categories, "Career-column" and "Career-feature" in 2016.
These articles were previously identified as "News" articles before 2016, as exemplified in the article "XX" [EXAMPLE].

> ### Gender Disparity Observed in Quotes

Reviewer: ```suggestion
### Quoted Speakers are More Often Male
```

This analysis relies upon accurate gender prediction of both authors and speakers.
To predict the gender of the speaker or author, we used the package genderizeR [@https://github.com/kalimu/genderizeR], an R package wrapper to access the genderize.io API [@https://genderize.io/] to get binary gender predictions for each identified first name.
We, unfortunately, cannot identify non-binary gender expression with the tools we used.
To identify our binary gender prediction error rate, we created a benchmark data set of thirty randomly selected news articles; ten from each of the following years: 2005, 2010, 2015.
In these articles, we hand-annotated the quoted speakers and their assumed gender.
We then compared our hand-annotated gender prediction against the gender prediction after the entire scraping, coreNLP quote annotator, and genderizeR pipeline.
Using our hand-annotated prediction as ground truth, we predicted a total of 55 female and 155 male observed names within the benchmark articles.
We found that the gender prediction had an accuracy of 0.914, with a Kappa statistic of 0.767 (Supplemental Figure 1 a).

> Now that we have established reasonable accuracy for our gender prediction, we determined the rate of gendered quotes within articles.

Reviewer: ```suggestion
After establishing a reasonable accuracy for gender prediction, we determined the rate of gendered quotes within articles.
```

> We first looked at the number of quotes identified within each news-related article (Figure @fig:fig2 b), totaling 120,309 quotes with a gender prediction for the speaker.

Reviewer: ```suggestion
We first examined the number of quotes identified within each news-related article (Figure @fig:fig2 b), totaling 120,309 quotes with a gender prediction for the speaker.
```

> As previously observed in article frequencies, we see quote frequencies vary by article type.

Reviewer: ```suggestion
Quote frequencies vary by article type.
```

The only article types in which there is a large discrepancy between the number of quotes and articles are "news-and-views" and "career-column.”
This discrepancy is expected in "news-and-views" since this article type typically is an overview of a specific scientific topic and rarely quotes authors.
Similarly, "career-column" is a perspective piece from a scientist who also rarely quotes other persons.

We then compared the number of male quotes to the number of male first and last authors published in Nature (Figure @fig:fig2 c).
We analyzed a total of 10466 first authors and 10494 last authors with a gender prediction.
As denoted by the red line, we find that both authorship and quotes are far from gender parity.
Additionally, we find a difference in author genders between first and last authors, with the last authors being more male-dominated.
Since Nature authorship may not represent scientific publishing as a whole, we also compared against a random selection of authors from English language journals published by Springer (Supplemental Figure 2 a).

> We observed a more significant gender gap between first and last authors in our selection of Springer articles; however, both first and last authors are much closer to parity than Nature authors.

Reviewer: ```suggestion
We observed a larger gender gap between first and last authors in our selection of Springer articles; however, both first and last authors are much closer to parity than Nature authors.
```

Reviewer: Nearly every journal now uses "significant" only for clauses that include a statistical test - I don't think (as written) we've done one for this difference.

Author: yes, this is true.

Figure @fig:fig3 a displays a schematic of the data processing and example names used in this analysis.
We divided our analysis into two parts: firstly, looking at the number of cited last authors with a specific name origin overall cited last authors in a particular year.
Secondly, looking at the proportion of quotes from a speaker with a specific name origin overall quotes in a particular year.
We used the proportion of last authors in Nature with a specific name origin overall Nature articles in a particular year for both analyses.
Additionally, in our supplemental analyses, we compare against the last authorship in a random selection of Springer articles.
We find that the number of quotes dramatically outnumbers the number of authors and citations (Supplemental Figure 3 a).
Still, the total number of observations per data type is larger than [XX] across all years.

In comparing the citation rate of last author name origins in news articles, we find most names are Celtic/English or European, both with a citation rate between 30-40% [MORE SPECIFIC] (Figure @fig:fig3 b).

> East Asian names are the highest proportion of cited names, with 10-20% cited last authorships [MORE SPECIFIC].

Reviewer: I don't understand how this aligns with the sentence above it.

Author: oops, missed a word. Fixed here: https://github.com/greenelab/nature_news_manuscript/pull/1/commits/61eb711d02938c47fee285c28a1a3d7ee005e00a

Panel D is a heatmap depicting the yearly difference in citation and mention rate for a specific country.
We only depict countries with a consistent and significant difference across all years.
Each cell contains the difference between citation and mention rates, with red denoting the lower difference between mention and citation and green a more considerable difference.
The left annotation bar titled "median_M_C" is the median difference across all observed years.
The "Class" annotation column denotes the binarized set definition of each country, either "Cited" or "Mentioned".
Panel E shows the top 15 words extracted from articles mentioning the countries depicted in Panel D, with the most significant proportional frequency between the two defined country sets.
The width of the bar depicts the log10(Frequency in Mentions / Frequency of Citations).
](https://github.com/nrosed/nature_news_disparities/raw/09df81eac6d8450127552f0b9dd3a6c1212b10eb/figure_notebooks/tmp_files/fig4_tmp/fig4_main.png "Figure 4"){#fig:fig4 width=6in}

> After identifying name origin differences in cited and quoted persons, we wanted to determine if countries 1) are represented at difference rates 2) vary in how scientific content is related to a country.

Reviewer: ```suggestion
After finding name origin differences between cited and quoted persons, we wanted to determine if news articles 1) represent countries at different rates 2) vary in the language used to describe scientific content related to each country.
```

Figure @fig:fig4 a shows example input data and a schematic of the analysis.
We provide further processing details in [Methods].
First, we interrogated the country affiliations of cited authors.
We assigned an affiliation to a manuscript if any author has affiliation from a specific country. 
Therefore a single manuscript may have multiple country affiliations.
We analyzed any possible country affiliation for an article due to limits imposed by the Springer API.
Affiliation query results from the Springer API return all country affiliations for a specific manuscript and are not linked to one particular author.
 
We analyzed a total of XX articles and considered all authors within the article and their affiliations for this analysis.
We found that most cited articles have at least one author with an affiliation within the United States, followed by the United Kingdom, Germany, and France (Figure @fig:fig4 b).
Interestingly, we found a strong citation over-enrichment of many top-cited countries, but we found no evidence of under-enrichment by East Asian countries (Figure @fig:fig4 c, Supplemental Figure 4 a).

> Next, we were interested in identifying content differences between countries or groups of countries.

Reviewer: ```suggestion
Next, we examined content differences between countries or groups of countries.
```

> For example, we wanted to identify if a country was the subject (i.e., their scientific policies, environment, pollution) or whether the science within a nation was the subject.

Reviewer: ```suggestion
For example, we wanted to determine the extent to which a country was the subject (i.e., their scientific policies, environment, pollution) or the research being performed within that country was the subject.
```

To do this, we need to calculate the mention rate of each country.

> To identify if a country was mentioned in an article, we took all organizations, countries, states, or provinces identified by coreNLP's named entity tagger.

Reviewer: ```suggestion
To identify if a country was mentioned in an article, we started with all organizations, countries, states, or provinces identified by coreNLP's named entity tagger.
```

> We then took all the identified region-related named entities and utilized OpenStreetMap to get the affiliated country.

Reviewer: ```suggestion
We then linked all the identified region-related named entities to countries with OpenStreetMap.
```

> Since there may be errors in both coreNLP and OpenStreetMap, a country is only assumed to be mentioned when at least two unique entities map to the same country in a single article.

Reviewer: ```suggestion
Since there may be errors in both coreNLP and OpenStreetMap, we only assumed a country was mentioned when at least two unique entities mapped to the same country in a single article.
```

On a benchmark set, this results in a class-balanced error of [XX] across all countries (Supplemental Figure 1 b).
 
 
Once we calculated the mention rate and used the previously described citation rate, we identified countries with a consistent skew towards either a higher or lower mention to citation rate (Figure @fig:fig4 d and Supplemental Figure 4 b).
This is defined as countries where the difference between citation and mention rates is in the top or bottom 5% per year.

> This outlier description allowed us to identify two sets of countries: high relative citation to mention rate: Germany, Netherlands, Japan, Spain, Denmark, Israel, Sweden, Switzerland, Belgium, Austria, China, South Korea, and Australia; low relative citation to mention rate: Canada, Democratic Republic of the Congo, Chile, Indonesia, Russia, South Africa, Mexico, Brasil, Philippines, Colombia,  and India.

Reviewer: ```suggestion
This outlier description allowed us to identify two sets of countries based on their citation and mention rates.
Those with a high relative citation to mention rate were: Germany, Netherlands, Japan, Spain, Denmark, Israel, Sweden, Switzerland, Belgium, Austria, China, South Korea, and Australia.
Those with a low relative citation to mention rate: Canada, Democratic Republic of the Congo, Chile, Indonesia, Russia, South Africa, Mexico, Brasil, Philippines, Colombia,  and India.
```

> We removed all countries that were in both the top or bottom 5% in different years, thus excluding the United States and the United Kingdom from consideration. [List all of them]

Reviewer: ```suggestion
We removed all countries that were in both the top or bottom 5% in different years, which excluded the United States and the United Kingdom from consideration. [List all of them]
```
